api/src/main/resources/sql/migration.sql

```diff
@@ -551,7 +551,7 @@ CREATE TABLE nrmn.site_ref (
     wave_exposure integer,
     slope integer,
     site_attribute jsonb,
-    is_active boolean
+    is_active boolean NOT NULL
```

Comment:
This also needs to be done in https://github.com/aodn/NRMN  - contains the master ddl for core tables.  But can address that separately.

Reply:
https://github.com/aodn/NRMN/blob/master/db-migrate/ddl/CreateTables.sql#L42

Reply:
> This also needs to be done in https://github.com/aodn/NRMN - contains the master ddl for core tables. But can address that separately.

Yes I've prepared a PR  to set all `is_active` attributes NOT NULL in the migration code